### PR TITLE
Add NetObserv CLI

### PR DESCRIPTION
**Pull request by Amoghrd**

### CODE REVIEW

**Review by memodi — CHANGES_REQUESTED** [review]
Can you add below under inputs?
```
    netobserv_network-observability-cli_ci:
      as:
      - registry.ci.openshift.org/netobserv/network-observability-cli:ci
```

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml [hunk 1]
```diff
@@ -59,15 +59,20 @@ images:
     ci_tests-private-baseui_4.18:
       as:
       - registry.ci.openshift.org/ci/tests-private-baseui:4.18
+    netobserv_network-observability-cli_ci:
```
Comment (memodi) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml:62]: This needs to be in test-private-newpr image stanza

**Review by memodi — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml (hunk 1 above)
Comment (Amoghrd) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml:62]: oh yeah my bad

**Review by memodi — APPROVED** [review]
/lgtm

**Pull request merged**